namer/namer.cc

```diff
@@ -1061,8 +1061,10 @@ class SymbolDefiner {
             symbol.data(ctx)->setSuperClass(core::Symbols::Net_Protocol());
         }
 
-        // Don't add locs for root; 1) they aren't useful and 2) it'll end up with O(files in project) locs!
-        if (symbol != core::Symbols::root()) {
+        // Don't add locs for <root> or <PackageRegistry>; 1) they aren't useful and 2) they'll end up with O(files in
+        // project) locs!
+        if (symbol != core::Symbols::root() && symbol != core::Symbols::PackageRegistry() &&
+            symbol.data(ctx)->owner != core::Symbols::PackageRegistry()) {
```

Comment:
Why don't we add locs for symbols which descend from PackageRegistry? That seems over-eager.

Reply:
Because all immediate descendants of `<PackageRegistry>` are completely synthetic.

Every packaged file's AST is now encapsulated in a `module <PackageRegistry>::Foo_Bar_Package`, and we don't want to track all of their locs.

Reply:
Ah i had mixed up `owner` for super class.